pkg/ddc/jindo/transform.go

```diff
@@ -553,9 +554,19 @@ func (e *JindoEngine) transformToken(runtime *datav1alpha1.JindoRuntime, value *
 }
 
 func (e *JindoEngine) allocatePorts(value *Jindo) error {
-	// For now, Jindo only needs two ports for master and client rpc respectively
-	expectedPortNum := 2
 
+	// if not usehostnetwork then use default port
+	if !value.UseHostNetwork {
+		value.Master.Port.Rpc = DEFAULT_MASTER_RPC_PORT
+		value.Worker.Port.Rpc = DEFAULT_WORKER_RPC_PORT
+		if value.Master.ReplicaCount == JINDO_HA_MASTERNUM {
+			value.Master.Port.Raft = DEFAULT_RAFT_RPC_PORT
+		}
+		return nil
```

Comment:
Did you test it? Can the port be released successfully?

Reply:
the ContainerNetwork port can be released automatically? I am not sure

Reply:
Could you test it? thanks.FYI @TrafalgarZZZ